deploy/deploy_citus_server.yml

```diff
@@ -68,6 +68,27 @@
         state: started
         enabled: yes
 
+    - name: Install git
+      yum:
+        name: git
+        state: present
+
+    - name: Install dependencies
+      yum:
+        name: "{{item}}"
+        state: present
+      with_items:
+        - virtualenv
+        - python-devel
+        - libevent-devel
+        - gcc
+        - gcc-c++
+        - kernel-devel
+        - libxslt-devel
+        - libffi-devel
+        - openssl-devel
+        - openldap-devel
```

Comment:
@stirlingv Might it make sense to put this into a new role for `psql_authnz` that we can eventually use to automate that deployment?  That's what all these dependencies are for, no?  Thanks!

Reply:
@tanderegg yup, that's what these are for. Makes a lot of cense to put it in a new role. I'll do that, rebase and force push to update this PR whilst keeping the commit history clean.

Reply:
@stirlingv Thanks, let me know when ready!

Reply:
@tanderegg I've added the dependencies to a psql-authnz role, but not the automation of the deployment of the script itself... I've created an issue so we remember to actually do it: https://github.com/cfpb/aurora/issues/371